x/liquidity/keeper/invariants.go

```diff
@@ -57,16 +57,19 @@ func LiquidityPoolsEscrowAmountInvariant(k Keeper) sdk.Invariant {
 // We should approach adding these invariant checks inside actual logics of deposit / withdraw / swap.
 
 var (
-	invariantCheckFlag = true                     // TODO: better way to handle below invariant checks?
-	diffThreshold      = sdk.NewDecWithPrec(2, 1) // 20%
+	invariantCheckFlag = true // TODO: better way to handle below invariant checks?
+	// For the coin amount less than coinAmountThreshold, high errorRate don't mean
+	// that the calculation logic was wrong, thus ignore it.
```

Comment:
Can you elaborate more on detail on this comment? My suggestion is to have why we have these thresholds and the reasoning behind 5% of `errorRateThreshold` if you think that is necessary.

Reply:
OK, I'll add more information in the comment.
For the `errorRateThreshold`, I just chose a random small number, and there's no special meaning of that number.
But with too low value like 1%, the code panics and we should increase `coinAmountThreshold`.
Do you have any suggestions on how to describing it?